web/packages/operations/src/alarm.ts

```diff
@@ -314,7 +331,8 @@ export const initializeAlarms = async () => {
             ],
             AlarmDescription: LatencyDashboard,
             AlarmActions: [BRIDGE_STALE_SNS_TOPIC],
-            ...alarmCommandSharedInput,
+            ComparisonOperator: "GreaterThanThreshold",
+            ...absoluteValueBreachingAlarmConfig,
```

Comment:
I assume `GreaterThanThreshold` applies to most scenarios, If that’s the case, it could be included in `absoluteValueBreachingAlarmConfig` as the default option.